pkg/account/validate.go

```diff
@@ -0,0 +1,57 @@
+package account
+
+import (
+	"errors"
+	"reflect"
+	"regexp"
+
+	"github.com/Optum/dce/pkg/model"
+	validation "github.com/go-ozzo/ozzo-validation"
+)
+
+// We don't use the internal errors package here because validation will rewrite it anyways
+// Just spit out errors and turn them into validation errors inside the appropriate functions
+
+var accountIDRule = []validation.Rule{
+	validation.Match(regexp.MustCompile("^[0-9]{12}$")).Error("must be a string with 12 digits"),
+}
+
+func isNil(value interface{}) error {
```

Comment:
May want to move some of these to a generic package so they can be shared with leases, usage or this account package needs to be generalized into a new package name.